src/pages/Login/components/LoginComponent/useLogin.ts

```diff
@@ -1,67 +1,59 @@
 import { useState, useCallback, useMemo } from 'react';
 
-import { useForm } from '../../../../components/Form/useForm';
+import { useSigninMutation } from '../../../../redux/api/authApi';
 
-import { authAPI } from '../../../../api/auth';
+import { useForm } from '../../../../components/Form/useForm';
 
-import { InputTypeEnum } from '../../../../components/Input';
 import { FieldsList } from '../../../../components/Form/types';
-import { UseLoginComponentProps } from './types';
+import { InputTypeEnum } from '../../../../components/Input';
 
-const fieldsList = [FieldsList.login, FieldsList.password];
+import { setValueToUseStateFactory } from '../../../../utils/setValueToUseStateFactory';
```

Comment:
❗
Давай обзовём поближе к семантике того, что происходит, потому что в такой имплементации полученные фабрики будут работать только с полями форм и с определённым форматом их представления в стейте
```suggestion
import { setFormFieldValueFactory } from '../../../../utils/setFormFieldValueFactory ';
```

Reply:
Исправил